src/application/config/routes.php

```diff
@@ -89,7 +89,7 @@
 $route['api/v1/availabilities']['get'] = 'api/v1/availabilities/get';
 
 $route['api/v2/appointments/(:any)']['get'] = 'api/v2/appointmentsv2/get/$1';
-$route['api/v2/appointments/(:any)']['put'] = 'api/v2/appointmentsv2/put/$1';
+$route['api/v2/appointments/(:any)']['put'] = 'api/v2/appointmentsv2/updateAppointmentByIdIntegrated/$1';
```

Comment:
no, please use the same uri "api/v2/appointmentsv2/put/$1"

Reply:
if there is no specific reason, dont introduce this new one.
please check the EAI-19 be careful:

> Need new API V2:
> 
> PUT /api/v2/appointments/:id_integrated